src/components/Transactions/TransactionContract.tsx

```diff
@@ -0,0 +1,194 @@
+import { ArrowDownIcon, ArrowUpIcon, CheckCircleIcon, WarningTwoIcon } from '@chakra-ui/icons'
+import { Box, Collapse, Flex, Link, SimpleGrid, Tag } from '@chakra-ui/react'
+import { chainAdapters } from '@shapeshiftoss/types'
+import dayjs from 'dayjs'
+import { useState } from 'react'
+import { FaStickyNote, FaThumbsUp } from 'react-icons/fa'
+import { Amount } from 'components/Amount/Amount'
+import { CircularProgress } from 'components/CircularProgress/CircularProgress'
+import { IconCircle } from 'components/IconCircle'
+import { MiddleEllipsis } from 'components/MiddleEllipsis/MiddleEllipsis'
+import { Row } from 'components/Row/Row'
+import { RawText, Text } from 'components/Text'
+import { TxDetails } from 'hooks/useTxDetails/useTxDetails'
+import { fromBaseUnit } from 'lib/math'
+
+export const TransactionContract = ({ txDetails }: { txDetails: TxDetails }) => {
+  const [isOpen, setIsOpen] = useState(false)
+  const toggleOpen = () => setIsOpen(!isOpen)
+
+  const toAddress = txDetails.ensTo || txDetails.to || undefined
+
+  const transactionIcon = (() => {
+    switch (txDetails.direction) {
+      case 'in-place':
+        return <FaThumbsUp />
+      case 'outbound':
+        return <ArrowUpIcon />
+      case 'inbound':
+        return <ArrowDownIcon color='green.500' />
+      default:
+        return <FaStickyNote />
+    }
+  })()
+
+  return (
+    <>
+      <Flex
+        alignItems='center'
+        flex={1}
+        justifyContent='space-between'
+        textAlign='left'
+        as='button'
+        w='full'
+        py={4}
+        onClick={toggleOpen}
+      >
+        <Flex alignItems='center' width='full'>
+          <IconCircle mr={3}>{transactionIcon}</IconCircle>
+
+          <Flex justifyContent='flex-start' flex={1} alignItems='center'>
+            <Box flex={1}>
+              <Text
+                fontWeight='bold'
+                overflow='hidden'
+                flex={1}
+                textOverflow='ellipsis'
+                maxWidth='60%'
+                lineHeight='1'
+                whiteSpace='nowrap'
+                mb={2}
+                translation={[
+                  `transactionRow.parser.${txDetails.tx.data?.parser}.${txDetails.tx.data?.method}`,
+                  { symbol: '' }
```

Comment:
There is no such interpolation in parser translations. I guess it can be removed?

Reply:
There is indeed such an interpolation, e.g. `transactionRow.parser.yearn.deposit` - unless I've misunderstood your comment?